src/ToolBox/superpmi/superpmi-shared/methodcontext.h

```diff
@@ -95,6 +95,13 @@ class MethodContext
         DWORD     B;
         DWORD     C;
     };
+    struct DLDDD
+    {
+        DWORDLONG A;
+        DWORD     B;
+        DWORD     C;
+        DWORD     D;
+    };
```

Comment:
I think we may now prefer adding Agnostic versions for new structs ... @sandreenko ?

Reply:
> I think we may now prefer adding Agnostic versions for new structs ... @sandreenko ?

Agree, it is better to create agnostic structs for them, look at `Agnostic_CORINFO_RESOLVED_TOKENin` and `Agnostic_CORINFO_RESOLVED_TOKENout` for example.

Reply:
Thanks, will change.